### PR TITLE
Fix: Permissions to admin pages

### LINKED ISSUE #95: Wallet Session is not well-preserved.
If the admin connects and signs out, it keeps showing the menu state as if it was an admin even disconnected.

image

### PR DESCRIPTION
**Pull request by crisgarner**
- Fixes AdminLayout
- Add AdminLayout to Applications, Voters, and Distribute pages.
- Added link to info page in any Layout.

Fixes #95

### CODE REVIEW

File: src/features/admin/components/InvalidAdmin.tsx [hunk 1]
```diff
@@ -0,0 +1,12 @@
+import { Markdown } from "~/components/ui/Markdown";
```
Comment (kittybest) [inline src/features/admin/components/InvalidAdmin.tsx:1]: I remember there's a hook called `isAdmin`, maybe is what you need?
https://github.com/privacy-scaling-explorations/maci-rpgf/blob/main/src/hooks/useIsAdmin.ts

could add this to protect the admin-only pages.

File: src/features/admin/components/InvalidAdmin.tsx [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+
+export function InvalidAdmin() {
+  return (
+    <div>
+      <Markdown>{`### Invalid Admin Account
+  Only admins can access this page.
+
+  `}</Markdown>
```
Comment (0xmad) [inline src/features/admin/components/InvalidAdmin.tsx:9]: I don't think we need markdown here

File: src/layouts/AdminLayout.tsx [hunk 3]
```diff
@@ -12,11 +12,14 @@ type Props = PropsWithChildren<
...
 >;
 export const AdminLayout = ({ children, ...props }: Props) => {
-  const { address } = useAccount();
-
-  if (config.admins.includes(address!)) {
-    return <div>Only admins can access this page</div>;
+  const { data } = useSession();
+  if (data && config.admins.includes(data.address as `0x${string}`)) {
+    return <Layout {...props}>{children}</Layout>;
   }
```
Comment (0xmad) [inline src/layouts/AdminLayout.tsx:18]: Let's use [useIsAdmin.ts](https://github.com/privacy-scaling-explorations/maci-rpgf/blob/main/src/hooks/useIsAdmin.ts) here

**Review by 0xmad — CHANGES_REQUESTED** [review]
@crisgarner, thanks, just left some comments.

File: src/features/admin/components/InvalidAdmin.tsx (hunk 2 above)
Comment (crisgarner) [reply src/features/admin/components/InvalidAdmin.tsx:9]: Using Markdown here as CSS is not configured to distinguish styles for headings or paragraphs, example using just HTML 
image

Comment (0xmad) [reply src/features/admin/components/InvalidAdmin.tsx:9]: There is a `Heading` component and you can use it with different styles.

**Review by 0xmad — APPROVED** [review]
@crisgarner thanks!

**Pull request merged**